packages/fluentui/react-bindings/src/hooks/useCSS.ts

```diff
@@ -0,0 +1,95 @@
+import { serializeStyles } from '@emotion/serialize';
+import { ICSSInJSStyle, ThemePrepared } from '@fluentui/styles';
+import cx from 'classnames';
+// @ts-ignore
+import _Stylis from 'stylis';
+// @ts-ignore No typings :(
+import focusVisiblePlugin from '@quid/stylis-plugin-focus-visible';
+// @ts-ignore No typings :(
+import rtlPlugin from 'stylis-plugin-rtl';
+
+import { useFluentContext } from '../context';
+
+//
+// Types
+//
+
+// Inline keyframe definitions are not supported by useCSS() hook
+export type UseCSSStyle = Omit<ICSSInJSStyle, 'animationName'> & { animationName?: string };
+export type UseCSSStyleInput = string | UseCSSStyle | ((theme: ThemePrepared) => UseCSSStyle);
+
+//
+// Definitions
+//
+
+const SPECIFICITY_CLASSNAME = 'fcss';
```

Comment:
I find the previous `.increase-specificity` more informative than `fcss`.  We could also consider `.use-css` or something as well.

Having been told to consider production size of the CSS, we could use `.css` which would still correlate to the `useCSS` call.